src/ripple/app/ledger/LedgerHistory.cpp

```diff
@@ -60,7 +60,7 @@ LedgerHistory::insert(
 
```

Comment:
We could also get rid of `make_lock` (src/ripple/basics/make_lock.h) as the only reason this function existed was to deduce the mutex type (which CTAD now does).

Reply:
Nice! Fixed in 7846c24db `[fold] Remove make_lock`